bitnami/ghost/templates/ingress.yaml

```diff
@@ -2,42 +2,38 @@
 apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
-  name: {{ template "ghost.fullname" . }}
-  labels:
-    app: "{{ template "ghost.fullname" . }}"
-    chart: "{{ template "ghost.chart" . }}"
-    release: {{ .Release.Name | quote }}
-    heritage: {{ .Release.Service | quote }}
-  annotations:
-    {{- if .Values.ingress.certManager }}
+  name: {{ include "ghost.fullname" . }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+  annotations: 
```

Comment:
Trailing space. There's an extension in vscode to automatically remove all trailing spaces when saving.

Reply:
done